packages/tools/src/utilities/planar/index.ts

```diff
@@ -2,17 +2,20 @@ import filterAnnotationsWithinSlice from './filterAnnotationsWithinSlice';
 import getWorldWidthAndHeightFromCorners from './getWorldWidthAndHeightFromCorners';
 import filterAnnotationsForDisplay from './filterAnnotationsForDisplay';
 import getPointInLineOfSightWithCriteria from './getPointInLineOfSightWithCriteria';
+import interpolateAnnotation from './interpolateAnnotation';
 
 export default {
   filterAnnotationsWithinSlice,
   getWorldWidthAndHeightFromCorners,
   filterAnnotationsForDisplay,
   getPointInLineOfSightWithCriteria,
+  interpolateAnnotation,
 };
 
 export {
   filterAnnotationsWithinSlice,
   getWorldWidthAndHeightFromCorners,
   filterAnnotationsForDisplay,
   getPointInLineOfSightWithCriteria,
+  interpolateAnnotation,
```

Comment:
I don't think it belongs here in the planar, since it is specific to the freehand as the annotation. Maybe we create the freehand name space and we can access this utility as `csTools.utilities.freeHandROITool.interpolatePoints/interpolateData`?

Reply:
ok I will change